manifests/v1beta1/components/katib-controller/katib-ui-virtual-service.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: katib-ui
+spec:
+  gateways:
+    - kubeflow-gateway
+  hosts:
+    - "*"
+  http:
+    - match:
+        - uri:
+            prefix: /katib/
+      rewrite:
+        uri: /katib/
+      route:
+        - destination:
+            host: katib-ui.$(katib-ui-namespace).svc.$(clusterDomain)
```

Comment:
Will we get these values from the appropriate stack.
For example: https://github.com/kubeflow/manifests/blob/master/distributions/stacks/gcp/kustomization.yaml ?

Reply:
Yes exactly! Let's keep those as they are and we can introduce changes afterwards if we find it's not a good pattern.